greedy_method/greedy_coin_change.py

```diff
@@ -0,0 +1,48 @@
+"""
+Test cases:
+find_minimum_change(987)
+500 100 100 100 100 50 20 10 50 2
+find_minimum_change(500)
+500
+find_minimum_change(0)
+The total value cannot be zero or negetive
```

Comment:
Change `negetive` -> `negative`. Codespell is pretty picky about misspellings, and the build will error.